### DISCUSSION TITLE
@avelino: could you please clarify status and your plans for this repo

### DISCUSSION THREAD

**Discussion by alexbozhenko** [avelino/awesome-go#5149] [General]
Hi, Avelino.

First of all - thanks for creating this repo!

I noticed that nearly half of the entries here are abandonware - have not been updated in the last year.
We discussed how there was an attempt to automate the flagging of such repos, which has been broken in the past several months.
https://github.com/avelino/awesome-go/discussions/4871#discussioncomment-7833931
I spent some time looking at it:
https://github.com/avelino/awesome-go/pull/5127

And fixing the test created this issue, flagging nearly half of all the repos:
https://github.com/avelino/awesome-go/issues/5136
It was disappointing to see that you closed the issue without saying a word.
![image](https://github.com/avelino/awesome-go/assets/5295986/b00ca025-9157-448e-affe-88e796b2f1b3)
I wouldn't agree that it is completed.

At it's current state, this repo is a listicle of some truly awesome projects, but with big number of some outdated garbage, which clearly no one will ever be interested in using.

Again, this repo is linked to from https://go.dev/wiki/.
> [Awesome Go](http://awesome-go.com/) is another great resource for Go programmers, curated by the Go community. 

If I were a newbie who just discovered go, who clicked on the link from the go wiki, only to find out that "awesome go list" is half outdated projects, I would make a conclusion that go community sucks...

I understand that we all have busy lives, and maybe things have changed for you, and you do not have any time to maintain it.
In that case - could you please consider transferring the ownership to someone who does?

cc: other [MAINTAINERS](https://github.com/avelino/awesome-go/blob/main/MAINTAINERS)
@dukex  @dmitshur  @matrixik  @joeybloggs @kirillDanshin @felipeweb @appleboy @cassiobotaro @jtemporal @ceriath @panjf2000 @phanirithvij

**Comment by kirillDanshin** [comment]
Hey Alex, thanks for cc'ing. While I am on the MAINTAINERS file, and while I do have a lot of spare time now, I don't have access to this repo for quite a while now. If the current team needs my help, I'm all for it, as long as I'd have the permissions to actually do the maintainance of the list.

**Comment by phanirithvij** [comment]
I've reopened the issue, I think we need to tackle this and sorting/filtering features on the website (another much needed and requested feature which improves awesome go) via GitHub actions.

**Comment by panjf2000** [comment]
I have no objection to cleaning out those shoddy repositories on `awesome-go`, but I do think we shouldn't simply categorize some repositories as shoddy just for being inactive. Some of them might be specific and limited tools that require no more features after fulfilling the needs.

In short, I think we should do the cleanup job under a more rational standard and guideline.

**Comment by ceriath** [comment]
I do agree, the current state of this repo isn't really what it should be. But i also agreen with @panjf2000, inactivity does not necessarily mean useless. Lots of tools/libs are feature complete, as long as they get some module updates now and then.
Finding a new and more useful standard for the purpose of this repository and cleaning them up afterwards is the way to go here. I honestly just don't really know where to start.

**Comment by griggsca91** [comment]
are there any tasks that need to be done that can be delegated out?  Interested in this repo and it's success, but would need guidance for areas that I could be able to contribute too.
